tokens/eth/0x4b96bf1feF93A216914fc843D81207A027ce52b3.json

```
@@ -0,0 +1,30 @@
+{
+		"symbol": "VUU",
+		"address": "0x4b96bf1feF93A216914fc843D81207A027ce52b3",
```

```suggestion
		"address": "0x2ecB13A8c458c379c4d9a7259e202De03c8F3D19",
```
also needs to be chaned for the filename

---

0x2ecB13A8c458c379c4d9a7259e202De03c8F3D19 is the address of another token, not ours ;-)

I think I fixed the address capitalization as part of 6b6083f, did i do it wrong?

thanks, appreciate it